### PR TITLE
Adding astra.hackclub.com

### PR DESCRIPTION
**Pull request by AVSCGREAT**
# [Adding/Deleting/Updating] `subdomain.[hackclub.com/dino.icu/etc]`

## Description

Did all required edits to club website.

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -2855,3 +2859,8 @@ syria:
   - ttl: 600
     type: CNAME
     value: hackclub-syria.github.io.
+    
+astra:
+  - ttl: 600
+    type: CNAME
+    value: astra-collective.github.io.
```
Comment (Muirrum) [inline hackclub.com.yaml:2866]: Delete this redundant entry.

**Review by Muirrum — CHANGES_REQUESTED** [review]

**Comment by Muirrum** [comment]
addtl -- what's the difference between this and #1249 ?

File: hackclub.com.yaml (hunk 1 above)
Comment (AVSCGREAT) [reply hackclub.com.yaml:2866]: ok...other two entries had probelems in it so i closed them.

Comment (AVSCGREAT) [reply hackclub.com.yaml:2866]: will make new correct PR

**Pull request closed without merging**